Gems/AtomLyIntegration/CommonFeatures/Code/Source/PostProcess/RadiusWeightModifier/EditorRadiusWeightModifierComponent.cpp

```diff
@@ -25,7 +25,7 @@ namespace AZ
                 if (AZ::EditContext* editContext = serializeContext->GetEditContext())
                 {
                     editContext->Class<EditorRadiusWeightModifierComponent>(
-                        "Radius Weight Modifier", "Modifies PostFX override factor based on proximity of an influencer against this entity's bounding sphere")
+                        "PostFX Radius Weight Modifier", "Modifies PostFX override factor based on proximity of an influencer against this entity's bounding sphere")
```

Comment:
Be careful as any Python script possibly using this component will stop working with this change.
I seem to remember that we added an Alias property for this purpose, let me double check...

Reply:
Yes. They're probably are pre-existing Python tests for these. But otherwise great change!

Reply:
Hum no, I might be misremembering. The `EditorComponentAPIComponent::FindComponentTypeIdsByEntityType` function explicitly checks the m_name property only.

```
if (componentClass->m_editData->m_name == componentTypeNames[i])
```

I would do a quick check in the python scripts to make sure it's not used and fix it if it is. Not sure if this could break compatibility with customer scripts though.

Reply:
Good call yes it is referenced in an atom hydra test so I will update that, thanks!